### DISCUSSION TITLE
2022-04-15 Agenda

### DISCUSSION THREAD

**Discussion by 7yl4r** [USF-IMARS/python-tech-workgroup#36] [General]
Please use this thread to discuss plans for this Friday's meeting. Below is my planned agenda. I don't have a teaching topic planned - please request/suggest something.

* github discussions usage - is this working? should we use something else?
* casual chat - I think we could benefit from a casual synchronous chat platform. eg gitter, whatsapp, discord, slack.
* project updates
    * l2-proc arctic work is ready for next steps on manglilloo
    * bb3 is on backburner
    * ...
* Tylar's python module setup cheatsheet needs updating/fixing

**Comment by 7yl4r** [comment]
attending: Dan, Carolina, Sebastian, Tylar
 
* Plan for discussions is to post the agenda to discussions & send a Friday morning reminder email.
* slack is the frontrunner for chat platform
    * we may want to jump onto an existing python discussion one 
    * we may want an IMaRS-only one for general lab discussion
* project updates
    * l2-proc arctic work is ready for next steps on manglilloo
    * bb3 is on backburner
    * carolina's new project is set up: https://github.com/cperaltab/OBISdataManagement

**Reply by dotis** [reply]
I'm not sure where to put this, but I have a question on quarto. In the ECOSTRESS workshop repo, the yml file is called "_quarto.yml". @7yl4r mentioned it last week and I saw an email about it. 

https://github.com/dotis/2022-ECOSTRESS-Cloud-Workshop/blob/main/_quarto.yml
